src/containers/Header/MobileMenu.jsx

```diff
@@ -98,6 +99,9 @@ class MobileMenu extends Component {
     if (MODE !== 'mainnet') {
       routes = removeRoutes(routes, 'tokens');
     }
+    if (!inNetwork) {
+      routes = removeRoutes(routes, 'explorer', 'network', 'tokens');
+    }
```

Comment:
there should also be a library method to return modified routes

Reply:
`removeRoutes` already is a library method